### PR TITLE
refactor document for new feature publishing

### PR DESCRIPTION
**Pull request by dengmingtong**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/en/deployment.md [hunk 1]
```diff
@@ -136,7 +136,9 @@ You can upload videos without captions for processing.
 
 2. Select the video, then select **Open**.
 
-3. Select the video language, and then choose **Start**. The system will start uploading the video and automatically process and generate captions.
+3. Select the video language, select custom vocabulary from **select a vocabulary** if you need, and then choose **Start**. The system will start uploading the video and automatically process and generate captions.
```
Comment (rrxie) [inline docs/en/deployment.md:139]: Same comment

**Review by rrxie — COMMENTED** [review]
We need to add the release changes to the revisions.md.

File: docs/en-branded/deployment.md [hunk 2]
```diff
@@ -136,7 +136,9 @@ You can upload videos without captions for processing.
 
 2. Select the video, then select **Open**.
 
-3. Select the video language, and then choose **Start**. The system will start uploading the video and automatically process and generate captions.
+3. Select the video language, select custom vocabulary from **select a vocabulary** if you need, and then choose **Start**. The system will start uploading the video and automatically process and generate captions.
```
[the remark this answers is not in the document]
Comment (dengmingtong) [reply docs/en-branded/deployment.md]: Add refer link for custom vocabulary

File: docs/en/deployment.md (hunk 1 above)
Comment (dengmingtong) [reply docs/en/deployment.md]: Add refer link for custom vocabulary

File: README.md [hunk 3]
```diff
@@ -96,7 +96,9 @@ You can upload videos without captions for processing.
 
 2. Select the video, then select **Open**.
 
-3. Select the video language, and then choose **Start**. The system will start uploading the video and automatically process and generate captions.
+3. Select the video language, select custom vocabulary from **select a vocabulary** if you need (Refer to [Custom Vocabulary][transcribe-custom-vocabulary] to learn more about custom vocabulary), and then choose **Start**. The system will start uploading the video and automatically process and generate captions.
```
Comment (rrxie) [inline README.md:99]: refer to 首字母小写

File: docs/en-branded/revisions.md [hunk 4]
```diff
@@ -1,3 +1,4 @@
 | Date          | Change            |
 |---------------|-------------------|
-| January 2022 | Intitial release of version 1.0.0 |
\ No newline at end of file
+| January 2022 | Intitial release of version 1.0.0 |
+| December 2022 | Released version 1.1.0 <br> 1. Support Arabic and Portuguese languages.<br> 2. Support custom vocabulary.<br> 3. Reduce noise to improve captions quality<br> 4. Reduce captions translation latency<br> |
```
Comment (rrxie) [inline docs/en-branded/revisions.md:4]: Support Arabic and Portuguese for the default caption processing language.

File: docs/zh-branded/revisions.md [hunk 5]
```diff
@@ -1,3 +1,4 @@
 | 日期       | 发布说明 |
 |----------|--------|
-| 2022年1月 | 首次发布版本1.0.0  |
\ No newline at end of file
+| 2022年1月 | 首次发布版本1.0.0  |
+| 2022年12月 | 发布版本 1.1.0 <br> 1. 支持阿拉伯语和葡萄牙语。<br> 2. 支持自定义词汇表。<br> 3. 通过降噪处理提高字幕质量。<br> 4. 降低字幕翻译延时<br> |
```
Comment (rrxie) [inline docs/zh-branded/revisions.md:4]: 默认字幕处理语言支持阿拉伯语和葡萄牙语。

**Review by rrxie — COMMENTED** [review]
Remember to change the date to December 2022.

File: README.md (hunk 3 above)
Comment (dengmingtong) [reply README.md:99]: fixed

File: docs/en-branded/revisions.md (hunk 4 above)
Comment (dengmingtong) [reply docs/en-branded/revisions.md:4]: Fixed

File: docs/zh-branded/revisions.md (hunk 5 above)
Comment (dengmingtong) [reply docs/zh-branded/revisions.md:4]: Fixed

**Review by rrxie — APPROVED** [review]

**Pull request merged**